### DISCUSSION TITLE
Welcome to venus-os_dbus-mqtt-battery Discussions!

### DISCUSSION THREAD

**Discussion by mr-manuel** [mr-manuel/venus-os_dbus-mqtt-battery#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DilipTD** [comment]
i Want to connect the Seplos battery via rs485 with this feature and how to connect

**Reply by mr-manuel** [reply]
For this driver you need the data in MQTT. For connection via RS485 use the `dbus-serialbattery` driver.

**Reply by DilipTD** [reply]
Can you help for get  data from Seplos battery and I have parallel system

**Reply by mr-manuel** [reply]
So you want to aggregate multiple batteries into a virtual battery?

**Reply by DilipTD** [reply]
> So you want to aggregate multiple batteries into a virtual battery?

Yes

**Reply by mr-manuel** [reply]
I opened a new discussion for this.

**Reply by DilipTD** [reply]
Waiting sir

**Reply by mr-manuel** [reply]
Check the other discussions

**Comment by bemilic** [comment]
Hi Manuel,
great job!

I just start using MultiPlus + venus-os.
Unfortunately, I have had a SENEC V3 system with 5kW storage for two years.
Now I would like to operate a MultiPlus 5000 + GobelPower storage system in addition to 
this system (since I would otherwise go into the basement with a huge hammer...).
Your venus-os_dbus-mqtt-battery works perfect on my raspberry. With NodeRed i read the values
from the SENEC system (also batterie) and can see it on the Portal (just for information).

Now to my question (perhaps unfounded??):
What does the Multiplus do if I hang the GobelPower battery including BMS on the MultiPlus/cerbo? Doesn't the virtual battery "irritate" the WR? How does the WR know which battery values he should use for charging? The Multiplus then has, for example, two completely different SOC of the batteries as information?
​

**Reply by mr-manuel** [reply]
You mean you connect the BMS directly and once over the driver? There is only one controlling BMS. You can select it under DVCC settings at the bottom.

Under the settings you can also change which device to use as a battery monitor. Only one battery monitor/controlling bms is the master.

**Reply by bemilic** [reply]
not entirely correct. I must have expressed myself unclearly.
A battery (GobePower) will be connected directly to the MultiPlus (i.e. battery + BMS in near future)
and i read the SENEC battery (from the Home V3) via NodeRed already on the venus os and use it to set the values in your venus-os_dbus-mqtt-battery. This means that 2 battery values will probably be displayed in the portal (I can already see the display of your driver in the portal at the moment. A bit of theory at the moment, as the MultiPlus won't be there until tomorrow ;-) ).
